circle.yml

```diff
@@ -12,7 +12,7 @@ dependencies:
   cache_directories:
     - testapp/node_modules
   post:
-    - ./node_modules/.bin/gulp format:enforce
+    - ./node_modules/.bin/gulp lint
```

Comment:
I think we still need the format check too, right? Also, let's move this down to before `npm test` so lint failures don't block test runs.